controlplane/kubeadm/api/v1alpha4/kubeadmcontrolplanetemplate_webhook.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha4
+
+import (
+	"reflect"
+
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+	ctrl "sigs.k8s.io/controller-runtime"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+)
+
+// log is for logging in this package.
+var kubeadmcontrolplanetemplatelog = logf.Log.WithName("kubeadmcontrolplanetemplate-resource")
```

Comment:
I know the loggers are auto-generated, but the info messages do not add anything. Plus IIRC, controller-runtime logs all the webhook requests it receives. So that is covered.

Reply:
@srm09 Afaik it doesn't log them on the default log level. But I wouldn't add log statements here. When we do it we should do it across the repo.